src/com/owncloud/android/operations/UploadFileOperation.java

```diff
@@ -372,6 +372,9 @@ protected RemoteOperationResult run(OwnCloudClient client) {
             if (temporalFile != null && !originalFile.equals(temporalFile)) {
                 temporalFile.delete();
             }
+            if (result == null){
+                return new RemoteOperationResult(false, 404, null);
```

Comment:
I do not know anymore.
Maybe it was just a placeholder while finding something better.
Do you have a suggestion?